frontend/src/Components/Pages/ManageProfilePage.js

```diff
@@ -22,60 +74,58 @@ function ManageProfilePage() {
+                            <Form.Control type="password" value={password} onChange={(e) => setPassword(e.target.value)} placeholder={password} />
                         </Form.Group>
 
                         <Form.Group className="mb-2" controlId="formBasicAddress">
                             <Form.Label>Your Home Address</Form.Label>
-                            <Form.Control type="name" placeholder="5825 Ave Eldridge, H4W2E3, Cote-Saint-Luc, Quebec, Canada" />
+                            <Form.Control type="home_address" value={home_address} onChange={(e) => setAddress(e.target.value)} placeholder={home_address} />
                         </Form.Group>
 
                         <Form.Group className="mb-2" controlId="formBasicNumber">
                             <Form.Label>Your Phone Number</Form.Label>
-                            <Form.Control type="name" placeholder="514-550-8326" />
+                            <Form.Control type="phone_number" value={phone_number} onChange={(e) => setPhone(e.target.value)} placeholder={phone_number} />
                         </Form.Group>
 
                         <Form.Group className="mb-2" controlId="formBasicGender">
                             <Form.Label>Gender</Form.Label>
-
-                            <Form.Select>
+                            <Form.Select value={gender} onChange={(e) => setGender(e.target.value)}>
                                 <option>Female</option>
                                 <option>Male</option>
                             </Form.Select>
                         </Form.Group>
 
                         <Form.Group className="mb-2" controlId="formBasicAccount">
                             <Form.Label>Account Type</Form.Label>
-
-                            <Form.Select>
+                            <Form.Select value={account_type} onChange={(e) => setAccountType(e.target.value)}>
                                 <option>Customer</option>
                                 <option>Seller</option>
                                 <option>Admin</option>
-                            </Form.Select>
+                             </Form.Select>
                         </Form.Group>
                        
                         <Form.Group className="mb-4" controlId="formBasicChange">
                         </Form.Group>
-
-                        <div className='d-grid'>
-                            <Button variant="primary" type="submit">
+                    </Form>
+                    <div className='d-grid'>
+                            <Button variant="primary" type="submit" onClick={handleSaveChanges}>
                                 Save Changes
                             </Button>   
                         </div>
-                    </Form>
                 </Card.Body>
             </Card>
+            <button onClick={() => console.log(handleSaveChanges) }>hello</button>
```

Comment:
What is this button again ? Is this the button to test something ?

Reply:
It was the test button. I am removing it now :)